bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AnomalousCouplings-SMP/WWTolnulnu_01j_aTGC_lep_WWmass_4f_NLO_FXFX_UL/WWTolnulnu_01j_aTGC_lep_WWmass-400to600_4f_NLO_FXFX_W_minus/WWTolnulnu_01j_aTGC_lep_WWmass-400to600_4f_NLO_FXFX_W_minus_madspin_card.dat

```diff
@@ -0,0 +1,37 @@
+#************************************************************
+#*                        MadSpin                           *
+#*                                                          *
+#*    P. Artoisenet, R. Frederix, R. Rietkerk, O. Mattelaer * 
+#*                                                          *
+#*    Part of the MadGraph5_aMC@NLO Framework:              *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+#
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#initialization parameters
+ set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+ set BW_cut 15                # cut on how far the particle can be off-shell
+ set max_weight_ps_point 400  # number of PS to estimate the maximum for each event
+# 
+
+set max_running_process 1
+
+define ell+ = e+ mu+ ta+
+define ell- = e- mu- ta-
+define V = w+ w-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+
+#dummy decay
+
+#decay w+ > ta+ vt
```

Comment:
Please clean up the unused/unnecessary lines

Reply:
done